examples/todo-list/src/__tests__/acceptance/todo-list.acceptance.ts

```diff
@@ -178,6 +190,20 @@ describe('TodoListApplication', () => {
       .expect(200, [toJSON(listInBlack)]);
   });
 
+  it('includes Todos in query result', async () => {
+    const list = await givenTodoListInstance(todoListRepo);
+    const todo = await givenTodoInstance(todoRepo, {todoListId: list.id});
+    const filter = JSON.stringify({include: [{relation: 'todos'}]});
```

Comment:
This is a rather complex way how to specify a simple inclusion. Can you please open a new issue to allow simpler syntax too? I believe LB3 allows `{include: ['todos']}`, we should support the same shortcut in LB4 too.

Please add the new issue to the epic https://github.com/strongloop/loopback-next/issues/1352 Inclusion of related models

Reply:
Created https://github.com/strongloop/loopback-next/issues/3205